Sources/Tentacle/Commit.swift

```diff
@@ -7,13 +7,19 @@
 //
 
 import Foundation
-import Argo
-import Curry
-import Runes
 
-public struct Commit {
+public struct Commit: ResourceType {
+    public struct Hash: Decodable {
+        let raw: String
+
+        public init(from decoder: Decoder) throws {
+            let container = try decoder.singleValueContainer()
+            self.raw = try container.decode(String.self)
+        }
+    }
+
     /// SHA of the commit
-    public let sha: SHA
+    public let sha: Hash
```

Comment:
Why did this change from a `SHA`?

Reply:
Because it’s not exactly the same type IIRC

Reply:
Can you elaborate on this? I think it should be the same thing.